common/helpers/field/reduce-add-another-fields.js

```diff
@@ -18,13 +18,35 @@ function reduceAddAnotherFields(allFields = {}, values = {}) {
         const name = `${prefix}[${itemField}]`
         // IDs cannot contain square brackets so they need to be removed
         const id = name.replace(/\[|\]/gi, '-')
+        const dataName = `${field.name}[%index%][${itemField}]`
+        const dataId = dataName.replace(/\[|\]/gi, '-')
         const opts = {
           // tell form wizard to not render field at top level
           skip: true,
           idPrefix: id,
           prefix,
           name,
           id,
+          ...(fieldTemplate.component !== 'govukCheckboxes' && {
```

Comment:
Is the checkboxes component the only one we want to exclude? Not sure I fully understand the purpose of this condition?